stream-chat-android-ui-common/src/test/kotlin/com/getstream/sdk/chat/viewmodel/MessageInputViewModelTest.kt

```diff
@@ -2,33 +2,7 @@
 
 package com.getstream.sdk.chat.viewmodel
 
-import androidx.lifecycle.MutableLiveData
```

Comment:
Tests will be refactored in a separate PR so that the Core team is not blocked for further migration.